runtime/build.gradle.kts

```diff
@@ -18,45 +16,80 @@ kotlin {
         nodejs {}
     }
 
+    val onPhone = System.getenv("SDK_NAME")?.startsWith("iphoneos")?:false
+    if(onPhone){
```

Comment:
On second thought, might be better to have iosArm64 and iosX64 separate for all situations... 🤔

Reply:
How does Kotlin Multiplatform Native work? Do we need to explicitly declare each architecture we intend to target? Or can we configure it to build for whatever architecture the build is running on?

We'd probably want to support Linux builds so we could run unit tests in CI (once we actually have some cross-platform unit tests...). But we can deal with that later.

Reply:
Yes we have to specify each platform. This way, KMP will generate the correct output for each platform.

Reply:
And for the Linux, it's pretty much adding `linux64()`